### DISCUSSION TITLE
Cannot create GeoDataFrame because of name 'CRS' not defined

### DISCUSSION THREAD

**Discussion by orietrans** [anitagraser/movingpandas#206] [Q&A] [answered]
Hi! I am a green hand to this package. I ran "0-debug.ipynb" in your GitHub repo.
I successfully installed all the dependencies and your MovingPandas package.
However, I met the NameError, shows that name 'CRS' is not defined. 
Could you please help me how to resolve this issue? I think many potential
users like me may face the similar issues, so it is necessary to solve this problem.
Looking for your further reply. Many thanks.
![image](https://user-images.githubusercontent.com/47074370/156976037-edc2be28-8353-44e5-b6f5-cb8ea72cf72d.png)
![image](https://user-images.githubusercontent.com/47074370/156976047-7a4aafdf-d7b3-4f28-9fb2-e775f65ced86.png)

**Comment by anitagraser** [comment, marked as answer]
Hi Li. Unfortunately I cannot reproduce this issue locally or on https://mybinder.org/v2/gh/anitagraser/movingpandas/master?filepath=tutorials/0-debug.ipynb. Please compare your system info output to the output seen on mybinder. It's most certainly a problem with the python environment.
